docs/source/en/using-diffusers/controlnet.md

```diff
@@ -467,18 +484,24 @@ canny_image = Image.fromarray(canny_image).resize((1024, 1024))
   </div>
 </div>
 
+You need to install controlnet_aux for the human pose estimation task:
+  
+```py
+# uncomment to install the necessary library in Colab
+#!pip install -q controlnet_aux
+```
+
 Prepare the human pose estimation conditioning:
 
 ```py
 from controlnet_aux import OpenposeDetector
-from diffusers.utils import load_image
-
-openpose = OpenposeDetector.from_pretrained("lllyasviel/ControlNet")
 
+original_openpose = OpenposeDetector.from_pretrained("lllyasviel/ControlNet")
```

Comment:
This section should be like below otherwise it won't work

```py
from controlnet_aux import OpenposeDetector

openpose = OpenposeDetector.from_pretrained("lllyasviel/ControlNet")
original_image = load_image(
    "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/person.png"
)
openpose_image = openpose(original_image)
make_image_grid([original_image, openpose_image], rows=1, cols=2)
```

Reply:
Yeah, there is something wrong here 😲. I don't know how I did this...